docs/roadmap.md

```diff
@@ -0,0 +1,98 @@
+# Roadmap
+
+## Fit and finish
+
+These updates are more along the lines of general usability
+
+| idea                            | description                                                | tracker | status        |
+|---------------------------------|------------------------------------------------------------|---------|---------------|
```

Comment:
Are these roughly in the order you think they should be implemented? That would be nice if they were.

Reply:
No let's just sort out priority when I open the Jiras.  I'm also curious on evengelist feedback to sort out which should be done first.

Reply:
OK